### PR TITLE
initialPR: Initial PR for Dashboard Screen

### PR DESCRIPTION
**Pull request by raynesmike**
This is to initiate our project that will contain the initial look for dashboard screen

### CODE REVIEW

**Review by lasharela — APPROVED** [review]

**Review by leo-he-1 — APPROVED** [review]

**Comment by Josetf2396** [comment]
Looks good

**Comment by Dulaya** [comment]
Looks good to me.

**Review by mshago — APPROVED** [review]
looks great

**Review by leo-he-1 — APPROVED** [review]
Good Changes

File: App.js [hunk 1]
```diff
@@ -1,112 +1,6 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
```
Comment (aahenry51) [inline App.js:4]: Nice Comment

File: App.js [hunk 2]
```diff
@@ -1,112 +1,6 @@
...
-  },
-  sectionTitle: {
-    fontSize: 24,
-    fontWeight: '600',
-  },
-  sectionDescription: {
-    marginTop: 8,
-    fontSize: 18,
-    fontWeight: '400',
-  },
```
Comment (Nap1400) [inline App.js:106]: So all of these are removed.

**Comment by carizzuti** [comment]
Looks good

**Review by Nap1400 — APPROVED** [review]

**Review by aahenry51 — APPROVED** [review]

**Review by Josetf2396 — APPROVED** [review]

**Pull request merged**